README.md

```diff
@@ -80,17 +81,17 @@ and the current compatibility of this feature as well as Scylla-specific extensi
 
 Documentation can be found in [./docs](./docs) and on the
 [wiki](https://github.com/scylladb/scylla/wiki). There is currently no clear
-definition of what goes where, so when looking for something be sure to check
+definition of what goes where, so when looking for something, be sure to check
```

Comment:
If you're doing such syntax fixes, please title the patch "copy-editing" or "minor fixes", etc., not "state we support current nd previous release".